src/components/navbar.tsx

```diff
@@ -128,12 +125,12 @@ export default function Navbar() {
                             </Button>
                         </SheetTrigger>
                         <SheetContent className='md:hidden'>
-                            <nav className='flex flex-col items-center justify-end gap-6 select'>
-                                <Link href='#about' passHref onClick={handleClick}><TypographyNavLink>About Us</TypographyNavLink></Link>
-                                <Link href='#hackathons' passHref onClick={handleClick}><TypographyNavLink >Hackathons</TypographyNavLink></Link>
-                                <Link href='#events' passHref onClick={handleClick}><TypographyNavLink >Events</TypographyNavLink></Link>
-                                <Link href='#resources' passHref onClick={handleClick}><TypographyNavLink >Resources</TypographyNavLink></Link>
-                                <Link href='#contact' passHref onClick={handleClick}><TypographyNavLink >Contact</TypographyNavLink></Link>
+                            <nav className='flex flex-col items-end justify-end gap-6 select pt-10'>
+                                <Link href='/riverhacks' passHref><TypographyH4 className='hover:text-background hover:bg-foreground hover:cover'>RiverHacks 2024</TypographyH4></Link>
+                                <Link href='#about' passHref onClick={handleClick}><TypographyH4 className='hover:text-background hover:bg-foreground hover:cover'>About Us</TypographyH4></Link>
+                                <Link href='#pastevents' passHref onClick={handleClick}><TypographyH4 className='hover:text-background hover:bg-foreground'>Past Events</TypographyH4></Link>
+                                <Link href='#resources' passHref onClick={handleClick}><TypographyH4 className='hover:text-background hover:bg-foreground'>Resources</TypographyH4></Link>
+                                <Link href='#contact' passHref onClick={handleClick}><TypographyH4 className='hover:text-background hover:bg-foreground'>Contact</TypographyH4></Link>
                             </nav>
```

Comment:
Merged two classes to use updated tailwindcss and to use next.js <Link> elements for improved performance